distributionviewer/api/models.py

```diff
@@ -6,6 +6,12 @@
 class Metric(models.Model):
     name = models.CharField(max_length=255, unique=True)
     description = models.TextField()
+    type = models.CharField(
+        max_length=1, choices=(('C', 'Categorical'), ('N', 'Numerical')),
```

Comment:
Yeah, I just needed this for the other side when aggregating the data. If they are useful for the front-end we can add them to the API.